_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -3862,7 +3862,7 @@ mfxStatus MfxHwH264Encode::CheckVideoParamQueryLike(
 
     if (par.mfx.RateControlMethod == MFX_RATECONTROL_QVBR)
     {
-        if (!CheckRange(extOpt3->QVBRQuality, 0, 51)) changed = true;
+        if (!CheckRange(extOpt3->QVBRQuality, 1, 51)) changed = true;
```

Comment:
@ogolikov, @takulako, please look.

Reply:
It is usual for MediaSDK API that 0 is default value. There is no error if application sets parameter to 0. It means that MediaSDK  sets this parameter itself (function SetDefaults defines default parameters, it is 26 for QVBRQuality).  
I checked documentation, there is no info about default value for QVBRQuality. But, I think, it should be corrected.